settings.gradle

```diff
@@ -7,6 +7,9 @@
 
 pluginManagement {
     plugins {
+        id "com.github.spotbugs-base" version "${spotbugsPluginVersion}"
+        id "com.github.johnrengelman.shadow" version "${shadowJarPluginVersion}"
+        id "de.undercouch.download" version "${downloadPluginVersion}"
         id "net.researchgate.release" version "${releasePluginVersion}"
         id "io.ballerina.plugin" version "${ballerinaGradlePluginVersion}"
```

Comment:
I think we can remove the spotbugs and download plugins. 

- Spotbugs is used to analyze Java code.
- Download plugin is used to download the checkstyle rule files that are needed for checkstyle validations, which is also for Java. 

Since both these are for Java code analysis, we can remove them here.